pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -247,6 +248,27 @@ func AllWorkersUpgraded(c client.Client, scaler scaler.Scaler, metricsClient met
 		return false, err
 	}
 
+	silenceActive, err := metricsClient.IsSilenceActive()
+	if err != nil {
+		return false, err
+	}
+
+	var nodeCompleteTime *metav1.Time
```

Comment:
If we get past `ok, err := nodesUpgraded(c, "worker", logger)` we always have a nodeCompleteTime I think.
So is the check just `if !silenceActive {`?